### DISCUSSION TITLE
not showing anything

### DISCUSSION THREAD

**Discussion by Dellcash** [community/community#6972] [Copilot Conversations]
i install and uninstall copilot few more time but it did not show any suggestions for me also it did not show in status bar

**Comment by davecheney** [comment]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by Dellcash** [reply]
no i dont use any of them

**Reply by davecheney** [reply]
I cannot see any connections by your token, I suggest that there is a network issue preventing copilot from talking to our servers. I think there is an updated extension coming soon which will be more vocal when it is having network issues.
